docs/design/mockups/backoffice/layout.md

```diff
@@ -0,0 +1,18 @@
+# Layout Mockup
+
+![Layout Mockup](./images/layout.png)
+
+This is the main layout that will be used for all other pages in Backoffice.
+We can see the header, which contains "ÀIRNEIS" and "BACKOFFICE".
+
+There is also a lateral bar which contains all the Backoffice navigations
+
+The first part of the lateral bar is the "Product Management":
+- Products, to see all products in a table and manage them.
+- Categories, to see all categories in a table and manage them.
+- Higlanders, to manage "The Highlanders of the moment" in a table and choose the order in which they appear.
+- Orders, to manage the orders in a table and manage them
```

Comment:
Can you use ": " instead of "," commas. 

Example : `Products: To see all products in a table and manage them. `

Reply:
Yes, I'm going to do it now.